phpBB/phpbb/console/application.php

```diff
@@ -115,6 +116,16 @@ public function doRun(InputInterface $input, OutputInterface $output)
 			return 0;
 		}
 
-		return parent::doRun($input, $output);
+		try
+		{
+			$result = parent::doRun($input, $output);
+		}
+		catch (exception_interface $e)
+		{
+			$message = call_user_func_array(array($this->language, 'lang'), array_merge(array($e->getMessage()), $e->get_parameters()));
+			throw new \RuntimeException($message, $e->getCode(), $e);
```

Comment:
one question: do we want to link to the previous exception? I think yes because otherwise it is hard to debug.
But this way the whole exception chain will be displayed... 

EDIT: one solution could be to embed the previous exception only if we have debug enabled.